src/main/java/au/id/micolous/metrodroid/key/ClassicStaticKeys.java

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
I don't think you meant to include this file in this PR.

Reply:
Fixed